### PR TITLE
Avoid lib symlink

### PR DESCRIPTION
**Pull request by nielsenko**
- Export realm_dart.dart from realm package

### CODE REVIEW

**Comment by nielsenko** [comment]
There should be no user visible change, so I have skipped the CHANGELOG entry.

File: flutter/realm_flutter/tests/lib [hunk 1]
```diff
@@ -1 +1 @@
-../lib
\ No newline at end of file
+../../../lib/
```
Comment (nirinchev) [inline flutter/realm_flutter/tests/lib:1]: I know this is not a new thing, but I'm curious why we need to symlink lib inside the tests app? AFAICT the test project takes a dependency on the `realm` package, so I would expect it to pull the lib source code from there.

**Review by nirinchev — APPROVED** [review]
Looks good. So now the realm package in pub.dev will have a dependency on realm_dart for the dart source code and just add some native sources that pull the binaries automatically rather than relying on an install command?

File: flutter/realm_flutter/tests/lib (hunk 1 above)
Comment (nielsenko) [reply flutter/realm_flutter/tests/lib:1]: Indeed.. I just wanted to do the test refactoring in a separate PR as to not drown this one in small import updates.

**Comment by nielsenko** [comment]
> Looks good. So now the realm package in pub.dev will have a dependency on realm_dart for the dart source code and just add some native sources that pull the binaries automatically rather than relying on an install command?

Yes, and we just re-export all that is exported by realm_dart. It should not make a difference to our users.

**Pull request merged**